.eslintrc.js

```diff
@@ -3,15 +3,15 @@ module.exports = {
     browser: false,
     node: true,
   },
-  plugins: ['node', 'prettier'],
-  extends: ['eslint:recommended', 'plugin:node/recommended', 'prettier'],
+  plugins: ["node", "prettier"],
+  extends: ["eslint:recommended", "plugin:node/recommended", "prettier"],
```

Comment:
Seems odd to have this differ from the rest of the project, I think it's doing this because of missing prettier config in the root

Reply:
confirm